src/main/compa/models/Friendship2.java

```diff
@@ -0,0 +1,86 @@
+package compa.models;
+
+import com.google.gson.annotations.Expose;
+import org.bson.types.ObjectId;
+import org.mongodb.morphia.annotations.*;
+
+import java.util.Date;
+
+@Entity(value = "friendship2", noClassnameStored = true)
+@Indexes({
+        // @Index(value = "login", fields = @Field("login"), unique = true),
+})
+public class Friendship2 {
+
+    @Embedded
+    public enum Status {
+        PENDING,
+        ACCEPTED,
+        REUSED,
+        BLOCKED,
+        BLOCKED_Asked,
+        BLOCKED_Asker
+    };
+
+
+    @Id
+    private ObjectId id;
+
+    @Reference
+    private User me;
+
+    @Expose
+    @Reference
+    private Friendship2 sister ;
+
+    private Status status;
+
+    private Date date;
+
+    public Friendship2(){}
+
+    public Friendship2(User me, User sister){
+        this.me = me;
+        this.status = Status.PENDING;
+        this.sister  = new Friendship2(sister,me, 0);
```

Comment:
You should create another constructor `Friendship2( User me, Friendship sister)`and use it here